includes/Admin/Admin_AJAX.php

```diff
@@ -278,6 +239,48 @@ public function run_checks() {
 		);
 	}
 
+
+	/**
+	 * Get the AJAX_Runner runner.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return AJAX_Runner
+	 */
+	protected function get_runner() {
+		$runner = Plugin_Request_Utility::get_runner();
+
+		if ( ! ( $runner instanceof AJAX_Runner ) ) {
+			$runner = new AJAX_Runner();
+		}
+
+		return $runner;
+	}
+
+	/**
+	 * Get array of checks to run.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	protected function get_checks() {
+		$checks = filter_input( INPUT_POST, 'checks', FILTER_DEFAULT, FILTER_FORCE_ARRAY );
```

Comment:
Similar function present in [AJAX_Runner::get_check_slugs_param()](https://github.com/10up/plugin-check/blob/trunk/includes/Checker/AJAX_Runner.php#L65-L74)